pkg/key/key.go

```diff
@@ -1,22 +1,109 @@
 package key
 
 import (
+	"context"
 	"fmt"
+	"io/ioutil"
+	"os"
 
-	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	chartv1 "github.com/giantswarm/apiextensions-application/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	ClusterNameLabel        = "cluster.x-k8s.io/cluster-name"
-	ClusterWatchFilterLabel = "cluster.x-k8s.io/watch-filter"
+	ClusterNameLabel = "cluster.x-k8s.io/cluster-name"
```

Comment:
You can use the one upstream https://github.com/kubernetes-sigs/cluster-api/blob/main/api/v1alpha3/common_types.go#L27